src/cascadia/CascadiaPackage/CascadiaPackage.wapproj

```diff
@@ -46,10 +49,15 @@
     <PRIResource Include="Resources\*\Resources.resw" />
     <PRIResource Include="Resources\Resources.resw" />
   </ItemGroup>
-  <PropertyGroup Condition="'$(WindowsTerminalReleaseBuild)'!='true'">
-    <!-- This is picked up by CascadiaResources.build.items. -->
+
+  <!-- This is picked up by CascadiaResources.build.items. -->
+  <PropertyGroup Condition="'$(WindowsTerminalBranding)'=='Dev'">

```

Comment:
same thing applies here. If you don't set Branding=Dev, and leave it blank, we need to use the dev icons.